examples/1.html

```diff
@@ -10,7 +10,6 @@
         background: #eee;
       }
       .box {
-        position: relative;
```

Comment:
Thanks for quick reply. This style already exists in `css/styles.css`. I think this is redundant and may cause confusion since in order to make the resizable component work, the parent position has to be 'relative'.